src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +62,25 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// content generated by function
+export function DynamicRowGeneration = (rowElementMaxCount, labelArray) => {
+  // todo: QUESTION - should I use generateLabelSpanContents before or after chunking?
+  // todo: QUESTION - should I apply use this function for the Applicant Information Section as well?
+  // todo: what happens when we have an odd number of elements in an array with an even maxRowElementsValue? do we need to generate a blank span?
```

Comment:
I don't think you'd have to for Application Details. The individual fields have a fixed width of `48rem`, and the row itself has `justify-content: space-between` css attribute. So if there's an incomplete row at the end (3 elements, 2 row max, end up with 1 element on the end), it'll still fill in the column from the left.

For anything greater than two columns (`maxRowElementsValue > 2`) you would need to add a blank section to ensure that columns in each row line up with each other.